ceno_zkvm/src/circuit_builder.rs

```diff
@@ -136,7 +139,6 @@ pub struct ConstraintSystem<E: ExtensionField> {
     pub chip_record_beta: Expression<E>,
 
     pub debug_map: HashMap<usize, Vec<Expression<E>>>,
```

Comment:
- the above TODO seems unnessesary 
- we can remove `lk_table_expressions_items_map` because we dont extract each `record` of lk table in mock_prover

Reply:
I've also removed the `lk_table_expressions_namespace_map` as `lk_expressions_namespace_map` has the same type.